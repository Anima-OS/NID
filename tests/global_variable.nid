
#include <stdio.h>

FILE* stdin;
FILE* stdout;

int some_random_integer;

