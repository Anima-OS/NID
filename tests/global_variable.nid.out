{ declarations: 
   [ { pragma: '#include <stdio.h>' },
     { variable: 
        { type: { name: 'FILE', pointer: '*' },
          attributes: {},
          name: 'stdin' } },
     { variable: 
        { type: { name: 'FILE', pointer: '*' },
          attributes: {},
          name: 'stdout' } },
     { variable: 
        { type: { name: 'int' },
          name: 'some_random_integer',
          attributes: {} } } ] }
